src/pages/apply/style.css

```diff
@@ -0,0 +1,39 @@
+.applyContainer h1,
+.applyContainer h3,
+.applyContainer p {
+  color: var(--dark-blue);
+}
+
+.applyContainer h2 {
+  color: var(--beige);
+}
+
+.titleText,
+.tswebpageText,
+.projecteduText,
+.cfswebpageText  {
+  padding-left: 80px;
+  padding-right: 50px;
+}
+
+.positionsText  {
+  padding-top: 50px;
+  padding-left: 80px;
+}
+
+.faqText {
+  padding-top: 0;
+  padding-left: 80px;
+  padding-right: 50px;
+}
+
+.positionsBubble1 {
+  position: absolute;
+  top: 430px;
+  z-index: -1;
+}
+
+.positionsBubble2 {
+  padding-top: 100px;
+  z-index: -1;
+}
```

Comment:
Somehow the postions of the bubbles are a little off on my screen. See screenshots below :) 
![grafik](https://user-images.githubusercontent.com/81356927/224482691-68af1513-2062-4f0d-9c91-4e7039cbc1a5.png)
![grafik](https://user-images.githubusercontent.com/81356927/224482701-ae219699-6646-4a0c-9c4a-4b8d17a634df.png)